docs/reference/method/MongoDBClient-listDatabaseNames.txt

```diff
@@ -0,0 +1,74 @@
+====================================
+MongoDB\\Client::listDatabaseNames()
+====================================
+
+.. versionadded:: 1.7
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Client::listDatabaseNames()
+
+   Returns names for all databases on the server.
+
+   .. code-block:: php
+
+      function listDatabaseNames(array $options = []): Generator
+
+   This method has the following parameters:
+
+   .. include:: /includes/apiargs/MongoDBClient-method-listDatabases-param.rst
+
+   The ``$options`` parameter supports the following options:
+
+   .. include:: /includes/apiargs/MongoDBClient-method-listDatabases-option.rst
+
+Return Values
+-------------
+
+A generator containing the name of each database on the server.
```

Comment:
Since this hasn't been changed yet, please see my other comment about only documenting a Traversable or Iterator. There may not be any reason to inform users that we're using Generators internally.